hil-test/tests/spi_full_duplex_dma_pcnt.rs

```diff
@@ -1,6 +1,6 @@
 //! SPI Full Duplex DMA ASYNC Test with PCNT readback.
 //!
-//! Folowing pins are used:
+//! Following pins are used:
 //! SCLK    GPIO0
 //! MOSI    GPIO3 / GPIO10 (esp32s3)
 //! PCNT    GPIO2 / GPIO9 (esp32s3)
```

Comment:
gone ❤️

Reply:
oh yeah, I thought it would be a good idea to test a 1200 line PR somehow, turns out CI isn't happy with me :D